packages/lazy-import/package.json

```diff
@@ -0,0 +1,29 @@
+{
+	"name": "@wordpress/lazy-import",
+	"version": "1.0.0-prerelease",
+	"description": "Lazily import a module, installing it automatically if missing.",
+	"author": "The WordPress Contributors",
+	"license": "GPL-2.0-or-later",
+	"keywords": [
+		"wordpress"
+	],
+	"homepage": "https://github.com/WordPress/gutenberg/tree/master/packages/lazy-import/README.md",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/WordPress/gutenberg.git",
+		"directory": "packages/lazy-import"
+	},
+	"bugs": {
+		"url": "https://github.com/WordPress/gutenberg/issues"
+	},
+	"main": "lib/index.js",
+	"types": "build-types",
+	"dependencies": {
+		"execa": "^4.0.2",
```

Comment:
Noting that in the root `package.json we have:

https://github.com/WordPress/gutenberg/blob/65a4196b9df769e8529672f19e64c224af27095e/package.json#L143

We should either use "^4.0.0" here or update all occurrences to require `4.0.2`.

Reply:
> We should either use "^4.0.0" here or update all occurrences to require `4.0.2`.

I guess I could update the other occurrences.

Reply:
> We should either use "^4.0.0" here or update all occurrences to require `4.0.2`.

Updated in c1f6a66053, 59e9df2b8a.